ngraph/test/type_prop/clamp.cpp

```diff
@@ -21,23 +21,88 @@
 using namespace std;
 using namespace ngraph;
 
-TEST(type_prop, fused_clamp)
+TEST(type_prop, clamp_basic_f32)
```

Comment:
Please consider if we can generalize this test suite to reuse it across all of the remaining activation ops. If you can generalize then reuse approach from #4941.

Reply:
I like the idea, but for this operation I don't think it is applicable as it need to specify `min` and `max` attributes in the op instantiation. We could create a test suite and group the activations ops which do not require attributes.